articles/azure-arc/kubernetes/azure-rbac.md

```diff
@@ -171,8 +171,9 @@ az connectedk8s enable-features -n <clusterName> -g <resourceGroupName> --featur
     1. Add the following specification under `volumes`:
     
         ```yml
+        not
```

Comment:
Thanks for fixing the indentation on hostPath, that looks good - but looks like this "not" line is an accidental insertion?